docs/guides/src/main/server/enabletls.adoc

```diff
@@ -0,0 +1,58 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+
+<@tmpl.guide
+title="Configure Keycloak to use TLS"
+summary="Learn how to use HTTPS/TLS with Keycloak"
+includedOptions="https.* http.enabled">
+
+Transport Layer Security (short: TLS) is crucial to exchange data over a secured channel. For production environments, you should never expose Keycloak endpoints through HTTP, as sensitive data is at the core of what Keycloak exchanges with other applications. In this guide you will learn how to configure Keycloak to use HTTPS/TLS.
+
+== Configure TLS in Keycloak
+Keycloak can be configured to load the needed certificate infrastructure either from a certificate / key file pair in PEM format or from an existing keystore file. When both alternatives are set up, the certificate / key file configuration takes precedence over the keystore configuration.
```

Comment:
"certificate / key file pair in PEM format" is a bit long and hard to read. Perhaps something like:

```suggestion
Keycloak can be configured to load the needed certificate infrastructure using files in PEM format or from a Java KeyStore. When both alternatives are set up, the PEM files takes precedence over the Java KeyStores.
```

Reply:
will change with the upcoming version 👍